pkg/kev/kev_reconcile_test.go

```diff
@@ -51,7 +51,7 @@ var _ = Describe("Reconcile", func() {
 		r := kev.NewRenderRunner(workingDir, kev.WithUI(kmd.NoOpUI()))
 		r.LoadProject()
 		manifest, mErr = r.Manifest().ReconcileConfig()
-		Expect(mErr).NotTo(HaveOccurred())
+		Expect(mErr).NotTo(HaveOccurred(), workingDir)
```

Comment:
why do we need `workingDir` here?

Reply:
because otherwise we don't know which workingDir is failing it just says that something is failing in this line, this way if it fails here you can easily pinpoint the test since this property is mutated in every test pretty much